gr-audio/lib/osx/osx_sink.cc

```diff
@@ -115,7 +115,7 @@ void osx_sink::setup()
             err_str += ": ";
             for (UInt32 nn = 0; nn < all_names.size(); ++nn) {
                 err_str += all_names[nn];
-                if (nn != all_names - 1) {
+                if (nn != all_names.size() - 1) { // size() was missing
```

Comment:
Please remove the comment. Yeah it's wrong -- no need to track our mistakes in the code, it's enough to track them in the git log.